docs/design/fabric-v2+/attestation-interface.md

```diff
@@ -0,0 +1,63 @@
+# Attestation Interface
+
+This document defines the interface for producing and verifying an attestation.
+
+
+## Producing an attestation
+
+This interface is only available inside an enclave.
+
+```
+int init_attestation(
+    uint8_t* params,
+    uint32_t params_length);
+
+```
+```
+int get_attestation(
+    uint8_t statement[32],
```

Comment:
Let's add a comment here what statement is. I guess this is statement ends up in the userdata of an sgx quote.

Reply:
Yes, I have added a comment.